examples/experimental/dagster-airlift/dagster_airlift/core/sensor/builder.py

```diff
@@ -65,10 +65,16 @@ def check_keys_for_asset_keys(
 
 
 def build_airflow_polling_sensor_defs(
-    airflow_data: AirflowDefinitionsData,
+    *,
+    mapped_defs: Definitions,
+    airflow_instance: AirflowInstance,
     event_transformer_fn: Optional[DagsterEventTransformerFn],
     minimum_interval_seconds: int = DEFAULT_AIRFLOW_SENSOR_INTERVAL_SECONDS,
 ) -> Definitions:
```

Comment:
at this point we should have a docblock explaining this. Not convinced that "mapped" is the right adjective for this top-level API